README.md

```diff
@@ -55,7 +55,7 @@ npm install -g @stoplight/spectral
 yarn global add @stoplight/spectral
 ```
 
-Once installed, to verify your OAS file with spectral execute `spectral lint PATH_TO_YOUR_OAS -r spectral.yml`
+Once installed, to verify your OAS file with spectral execute `spectral lint PATH_TO_YOUR_OAS` having the `.spectral.yaml` file inside the directory from which you are calling the command.
```

Comment:
`PATH_TO_YOUR_OAS` --> `<oas-file>` 

Also I doubt that people will have the current directory as the dir where the guidelines (`.spectral.yaml` ) is – maybe we should keep the `-r api-guidelines/.spectral` there for better illustration (= being more explicit)

Reply:
The path to your oas is fixed :) 

As for the other one - I'll think about packaging guidelines.

Reply:
done